examples/file-feed/complete/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "file-feed",
```

Comment:
@diasdavid I'm totally fine changing the name! However, let's catch up on IRC today to discuss the name as I think we can find something more descriptive and less "loaded" word than "feed" (people will make assumption about what a fedd is that I think we can't provide in this example).